package.json

```diff
@@ -19,12 +13,31 @@
   "homepage": "https://bitbucket.org/vulpemventures/vulpem.com#readme",
   "devDependencies": {
     "@node-sass/node-module-importer": "^1.2.3",
+    "bulma": "^0.7.1",
     "http-server": "^0.11.1",
     "node-sass": "^4.9.2",
-    "nodemon": "^1.18.1",
+    "nodemon": "^1.18.3",
```

Comment:
if not needed anymore, remove old dependencies used by node scripts, like http-server, node-sass, nodemon, parcel-bundler.

Please leave there `push-dir` used for deploy

Reply:
fixed